.github/workflows/labeler.yml

```diff
@@ -0,0 +1,24 @@
+name: "Pull Request Labeler"
+on:
+- pull_request_target
```

Comment:
I _think_ this is the reason it's not running yet.

https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#pull_request_target

`pull_request_target` runs from the base context of the PR rather than the merge commit, but it's recommended for things like this so we may need to merge it to see if it works.